notifications_utils/recipients.py

```diff
@@ -58,6 +58,7 @@ def __init__(
         remaining_messages=sys.maxsize,
         allow_international_sms=False,
         allow_international_letters=False,
+        should_validate=True,
```

Comment:
I don't have any particular objections to the naming you chose of `should_validate`, other options off the top of my head are

`validate`
`validate_rows_and_cells`

but I don't mind really. Would be happy to stick with what you've got